src/styles/blocks/main.scss

```diff
@@ -0,0 +1,27 @@
+.catalog {
+  --column-count: 1;
+
+  padding: 50px 40px;
+  display: grid;
+  grid-template-columns: repeat(var(--column-count), $product-card-width);
```

Comment:
Ensure that the `$product-card-width` SCSS variable is defined in your variables file. If it's not defined, it could lead to compilation errors or unexpected behavior.